cppForSwig/Signer/ResolverFeed.h

```diff
@@ -17,7 +17,7 @@
 #include "../SecureBinaryData.h"
 #include "../Wallets/BIP32_Node.h"
 
-#include "../protobuf/Signer.pb.h"
+#include "protobuf/Signer.pb.h"
```

Comment:
Is this change necessary? It may break the autotools build

Reply:
Yes, otherwise it can't find the Signer.pb.h. Overall it's a good idea to get rid of all relative paths (`../`) for includes - I'd better fix the autotools builds (by adding appropriate include dirs) once approach the libraries separation PRs.